dist/jsoneditor.js

```diff
@@ -1715,7 +1715,15 @@ JSONEditor.AbstractEditor = Class.extend({
     return null;
   },
   getTitle: function() {
-    return this.schema.title || this.key;
+      //Title collapsable form sections
+      if (this.hasOwnProperty("hide_add_button")){
```

Comment:
Minor styling thing: space between closing parentheses and opening brace (here and elsewhere)